SwiftTalk/Views/Loader.swift

```diff
@@ -107,12 +116,14 @@ struct Loader: View {
     @State var isAnimating = false
```

Comment:
isn't this variable redundant? (don't see it being used anywhere)

Reply:
Yes =)